README.md

```diff
@@ -907,7 +908,28 @@ may interfere with the intentions of your use of various `openshift.with...` dir
 In the case where you run Jenkins out of an OpenShift pod though via the OpenShift Jenkins image, the 
 environment is set up such that conflicts of this nature will not occur.
  
+### Moving Images Cluster to Cluster
 
+Multiple OpenShift clusters is a common practice. A non-production and production cluster is very common. In a continuous integration pipeline, we want to achieve promotion to all environments and clusters. We need a way to promote images from 
+cluster to cluster. The `oc` command line tool has a built-in command, `image mirror`, that will promote images from
+cluster to cluster.
+
+```
+stage('Move Image') {
+	steps {
+    	withDockerRegistry([credentialsId: "source-credentials", url: "source-registry-url"]) {
```

Comment:
does our image already include the docker pipeline plugin needed for this to work?

Reply:
@bparees I just tested `withDockerRegistry` in minishift. The pipeline succeeded using the default Jenkins 2 image. I am not sure how backwards compatible this solution will be... I can change to `docker.withRegistry` if necessary. Thanks!

Here is my Jenkinfile:

```pipeline{
    
    agent { label 'maven' }
    stages {
        stage('Move Image') {
        	steps {
            	withDockerRegistry([credentialsId: "myproject-source-credentials", url: "https://registry.hub.docker.com"]) {
                    	sh """
                        	echo works
                      	"""
                 }
             }
        }
    }
}```

Reply:
since it's just documentation, i'm not overly concerned about backwards compatibility.